app/src/main/java/org/gdg/frisbee/android/activity/ArrowActivity.java

```diff
@@ -403,24 +384,20 @@ public void run() {
         }
     };
 
-    private void setQrCode() {
-        mHandler.post(updateQrCode);
-
-    }
-
     public long getNow() {
         return DateTime.now(DateTimeZone.UTC).getMillis();
     }
 
     private class BaseArrowHandler {
-        public void enablePush() {}
+        public void enablePush() {
+        }
 
         public void disablePush() {
         }
     }
 
-@TargetApi(Build.VERSION_CODES.ICE_CREAM_SANDWICH)
-    private class NfcArrowHandler extends BaseArrowHandler implements NfcAdapter.OnNdefPushCompleteCallback, NfcAdapter.CreateNdefMessageCallback{
+    @TargetApi(Build.VERSION_CODES.ICE_CREAM_SANDWICH)
```

Comment:
Oh, I've searched for API level checks but forgot to search for these. These should be removed now.